iOS/Chameleon/source/Chameleon/ApiDefinition.cs

```diff
@@ -16,524 +16,591 @@
-//	{
-//		// @property (readwrite, nonatomic) NSUInteger count;
-//		[Export ("count")]
-//		nuint GetCount ();
-//	
-//		// @property (readwrite, nonatomic) NSUInteger count;
-//		[Export ("setCount:")]
-//		void SetCount (nuint value);
-//	
-//		// +(UIColor *)colorFromImage:(UIImage *)image atPoint:(CGPoint)point;
-//		[Static]
-//		[Export ("colorFromImage:atPoint:")]
-//		UIColor ColorFromImage (UIImage image, CGPoint point);
-//
-//		// -(UIColor *)colorWithMinimumSaturation:(CGFloat)saturation;
-//		[Export ("colorWithMinimumSaturation:")]
-//		UIColor ColorWithMinimumSaturation (nfloat saturation);
-//
-//		// -(BOOL)isDistinct:(UIColor *)color;
-//		[Export ("isDistinct:")]
-//		bool IsDistinct (UIColor color);
-//
-//		// -(BOOL)getValueForX:(CGFloat *)X valueForY:(CGFloat *)Y valueForZ:(CGFloat *)Z alpha:(CGFloat *)alpha;
-//		[Export ("getValueForX:valueForY:valueForZ:alpha:")]
-//		bool GetValueForX (out nfloat X, out nfloat Y, out nfloat Z, out nfloat alpha);
-//
-//		// -(BOOL)getLightness:(CGFloat *)L valueForA:(CGFloat *)A valueForB:(CGFloat *)B alpha:(CGFloat *)alpha;
-//		[Export ("getLightness:valueForA:valueForB:alpha:")]
-//		bool GetLightness (out nfloat L, out nfloat A, out nfloat B, out nfloat alpha);
-//	}
-
-//	// @interface ChameleonPrivate (UIImage)
-//	[Category]
-//	[BaseType (typeof(UIImage))]
-//	interface UIImage_ChameleonPrivate
-//	{
-//		// +(UIImage *)imageWithImage:(UIImage *)image scaledToSize:(CGSize)newSize;
-//		[Static]
-//		[Export ("imageWithImage:scaledToSize:")]
-//		UIImage ImageWithImage (UIImage image, CGSize newSize);
-//
-//		// -(UIImage *)imageScaledToSize:(CGSize)newSize;
-//		[Export ("imageScaledToSize:")]
-//		UIImage ImageScaledToSize (CGSize newSize);
-//	}
-
-//	// @interface ChameleonPrivate (UIView)
-//	[Category]
-//	[BaseType (typeof(UIView))]
-//	interface UIView_ChameleonPrivate
-//	{
-//		// -(BOOL)isTopViewInWindow;
-//		[Export ("isTopViewInWindow")]
-//		bool IsTopViewInWindow ();
-//
-//		// -(UIView *)findTopMostViewForPoint:(CGPoint)point;
-//		[Export ("findTopMostViewForPoint:")]
-//		UIView FindTopMostViewForPoint (CGPoint point);
-//	}
+    [Static]
+    partial interface Constants
+    {
+        // extern double ChameleonVersionNumber;
+        [Field("ChameleonVersionNumber", "__Internal")]
+        double ChameleonVersionNumber { get; }
+
+        // extern const unsigned char [] ChameleonVersionString;
+        [Field("ChameleonVersionString", "__Internal")]
+        IntPtr ChameleonVersionString { get; }
+
+        // extern const UIStatusBarStyle UIStatusBarStyleContrast;
+        [Field("UIStatusBarStyleContrast", "__Internal")]
+        UIStatusBarStyle UIStatusBarStyleContrast { get; }
+    }
+
+    // @interface ChameleonConstants : NSObject
+    [BaseType(typeof(NSObject))]
+    interface ChameleonConstants
+    {
+    }
+
+    // @interface Chameleon (UIColor)
+    [Category]
+    [BaseType(typeof(UIColor))]
+    interface ChameleonColor
```

Comment:
This is incorrectly bound. There cannot be properties in categories. Also, binding static stuff inside categories is a bad idea due the only way to access them is through `(null as Foo).FooMethod ()`. What I recommend is to leave the class as before ([1][1] and [2][2]) then, remove the `BaseType` due is not necessary.

Also, there is a space between `Export` attribute and the property that must be deleted.

[1]: https://github.com/adlair/XamarinComponents/blob/4bfb9e9ff5d2a42b827c3ce80acb523ff3c63eec/iOS/Chameleon/source/Chameleon/ApiDefinition.cs#L56-L58
[2]: https://github.com/adlair/XamarinComponents/blob/4bfb9e9ff5d2a42b827c3ce80acb523ff3c63eec/iOS/Chameleon/source/Chameleon/Additions.cs#L14